lib/ex_unit/lib/ex_unit/case.ex

```diff
@@ -532,15 +550,125 @@ defmodule ExUnit.Case do
       end
 
   """
+  @spec register_attribute(env, name :: atom, opts :: keyword) :: :ok
```

Comment:
Minor nitpick: these type annotations feel a bit redundant having the function definition right below 🙂 

```suggestion
  @spec register_attribute(env, atom, keyword) :: :ok
```

Reply:
Maybe this was for an old version of ExDoc, but I thought if you didn't provide a name for the type, it got named `arg1`, `arg2`, etc. automatically. But, I'm not seeing that now. I'll remove them.